src/vs/editor/common/services/modelService.ts

```diff
@@ -195,10 +195,7 @@ export class ModelService extends Disposable implements IModelService {
 		if (config.editor && typeof config.editor.indentSize !== 'undefined' && config.editor.indentSize !== 'tabSize') {
 			const parsedIndentSize = parseInt(config.editor.indentSize, 10);
 			if (!isNaN(parsedIndentSize)) {
-				indentSize = parsedIndentSize;
-			}
-			if (indentSize < 1) {
-				indentSize = 1;
+				indentSize = Math.max(parsedIndentSize, 1);
```

Comment:
Is there a value to default to if it _is_ NaN?

Reply:
I think this matches what it was doing before. If it is NaN, then indent size is `"tabSize"`. `"tabSize" < 1` evaluates to false, so the conditional I removed would never be run